From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Subbu Vakati <subbusrv@amazon.com>
Date: Thu, 22 Oct 2026 00:23:38 +0000
Subject: [PATCH 3/4] Inline DRA GPU label in GPU processor test

The gce cloud provider is removed in EKS Anywhere builds, so the test
can't import it for the gce.DraGPULabel constant. Without this, go mod
tidy fails after the cloud providers are removed.
---
 .../processors/customresources/gpu_processor_test.go          | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cluster-autoscaler/processors/customresources/gpu_processor_test.go b/cluster-autoscaler/processors/customresources/gpu_processor_test.go
index fb96e4d430c7..782303f6b584 100644
--- a/cluster-autoscaler/processors/customresources/gpu_processor_test.go
+++ b/cluster-autoscaler/processors/customresources/gpu_processor_test.go
@@ -25,7 +25,6 @@ import (
 	apiv1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider/gce"
 	testprovider "k8s.io/autoscaler/cluster-autoscaler/cloudprovider/test"
 	"k8s.io/autoscaler/cluster-autoscaler/context"
 	"k8s.io/autoscaler/cluster-autoscaler/utils/gpu"
@@ -157,7 +156,8 @@ func TestFilterOutNodesWithUnreadyResources(t *testing.T) {
 		ObjectMeta: metav1.ObjectMeta{
 			Name: "nodeGPUViaDra",
 			Labels: map[string]string{
-				gce.DraGPULabel: "true",
+				// Value of gce.DraGPULabel, inlined because the gce cloud provider is removed in EKS Anywhere builds
+				"cloud.google.com/gke-gpu-dra-driver": "true",
 			},
 			CreationTimestamp: metav1.NewTime(start),
 		},
